src/views/tutorial-view/components/collection-card/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Collection as ClientCollection } from 'lib/learn-client/types'
```

Comment:
Should this component live in the main `src/components` directory? Or do you have plans for that already? I can see it being super reusable!

Reply:
Ditto, can also see it being re-usable! For now, it' just used in one place, so I figure better not to abstract it earlier than necessary.